ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -278,6 +278,8 @@ then
 		pwm disarmed -c 4 -p p:PWM_AUX_DIS4 -d ${OUTPUT_AUX_DEV}
 		pwm disarmed -c 5 -p p:PWM_AUX_DIS5 -d ${OUTPUT_AUX_DEV}
 		pwm disarmed -c 6 -p p:PWM_AUX_DIS6 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 7 -p p:PWM_AUX_DIS7 -d ${OUTPUT_AUX_DEV}
```

Comment:
That will work but might print an error message. Have you checked the boot output after applying the change?

Reply:
Thanks @LorenzMeier ,  You might be correct, with my (pixracer) hardware I don't observe impact on startup script output.  Please let me know if I am missing a detail.  I checked pwm_params_aux.c, and all 8 channels are defined in that file, so it seemed like an omission in rc.interface.  For convenience, here a comparison between PX4/Firmware master branch and this PR, and again, if I am overlooking something please let me know and I will update this PR!

![image](https://user-images.githubusercontent.com/2497951/43284730-f667d194-90d9-11e8-9188-f6b897d0bd9b.png)
![image](https://user-images.githubusercontent.com/2497951/43284756-0ae88b9a-90da-11e8-9144-fcde4b125603.png)

Reply:
I didn't do this initially because I wanted to find a solution that prevented parameters for non-existent hardware (eg pixracer PWM7 and PWM8) from being marked active.